rpi5.yaml

```diff
@@ -0,0 +1,260 @@
+desc: "Xen for Raspberry 5 comprehended"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/zephyr-dom0-xt"
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_BUILD_DIR: "build"
+
+  DOMD_BUILD_DIR: "build/domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "image"
+
+  ZEPHYR_DOMD_MACHINE: "rpi_5_xen_domd"
+  ZEPHYR_DOMD_TARGET: "zephyr/samples/basic/blinky"
+  ZEPHYR_DOMD_DIR: "zephyr"
+  ZEPHYR_DOMD_BUILD_DIR: "build_d"
+
+  ZEPHYR_DOMU_MACHINE: "xenvm"
```

Comment:
we do not suppose to have domu in target architecture. 
which should be: - d0less, Zephyr-> ctrl domain, Domd- Linux rpi5

Reply:
Deleted

Reply:
Obsolete